src/routing/Page.tsx

```diff
@@ -12,11 +12,20 @@ import { RouterContext } from './RouterContext';
 export default function Page(props: PageProps): JSX.Element {
   const { values, submitForm } = useFormikContext();
   const formValues = values as IFormData;
-  const listOfRoutes = useContext(RouterContext).listOfRoutes;
+  const { listOfRoutes, updateRoute } = useContext(RouterContext);
   const currentLocation = useLocation();
 
+  // const history = useHistory();
```

Comment:
do you want to keep this here for future use or can we remove it?

Reply:
I created a ticket to capture this [here](https://app.zenhub.com/workspaces/forms-library---platform-spike-team-61b0ae1f2cd3c30014e8a5b0/issues/department-of-veterans-affairs/va-forms-system-core/349)